app/views/results/common/_submission_selector.html.erb

```diff
@@ -24,8 +24,8 @@
     |
     (
     <em><%= t('marker.marks.total_mark') %></em>
-    <span id='current_mark_div'><%= result.total_mark %></span> /
-    <span id='total_mark_div'><%= assignment.total_mark %></span>
+    <span id='current_mark_div'><%= result.total_mark %></span>
+    <span> / <%= assignment.max_mark %></span>
```

Comment:
I changed it to keep consistency across files, since whenever we show the total we do it this way. However, I can revert this change.